src/screens/home/HistoriesScreen/components/TransactionItem.tsx

```diff
@@ -30,7 +27,7 @@ const TransactionItem = ({onPress, value}: Props) => {
                         <Text style={[textStyles.Body2, {
                             color: colors.c828489,
                             marginTop: scale(4),
-                        }]}>{value.timestamp ? (toFormattedDate(value.timestamp, defaultLocale)) : ''}</Text>
+                        }]}>{value.timestamp ? (toFormattedDate(value.timestamp, Config.defaultLocale)) : ''}</Text>
```

Comment:
can set default value for locales in toFormattedDate to Config.defaultLocale

Reply:
Updated all in this branch.

Reply:
<img width="1327" alt="Screen Shot 2022-03-03 at 17 21 19" src="https://user-images.githubusercontent.com/34257375/156545311-6f8d5c8c-adb1-4ef1-98e8-2674de80309c.png">